config.py

```diff
@@ -759,6 +759,12 @@
     ```
 """
 
+epochs_overlap: Optional[float] = None
```

Comment:
I think we should have two parameters:
one for the duration of the epochs, and one for the overlap between them:

```
fixed_length_epochs_duration
fixed_length_epochs_overlap
```
WDYT?

Reply:
Good idea, that would be better and less confusion for the creation of the events for rest tasks.